invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/GenerateForm.js

```diff
@@ -170,10 +170,10 @@ const mapFormFields = (obj, parentField, isCreate, formFieldsConfig, dropDumpOnl
     const rows = formFieldsConfig[fieldName]?.rows || fieldSchema?.metadata?.rows;
     if (fieldSchema.type === "string" && rows) {
       return (
-        <TextArea
+        <RichInput
```

Comment:
I think this is going to change it for all fields in the admin which are text areas, which may not necessarily allow rich text input?

Reply:
we could change the [mapping](https://github.com/inveniosoftware/invenio-administration/blob/63a92ebcace9e7c98bb1ffdf2f15390aff89e97e/invenio_administration/marshmallow_utils.py#L53) of SanitizedHTML to "html" rather than "string" so that we could handle it more specifically (with RichInput) and we could still keep the TextArea for strings with rows.

with @slint